cmd/kubeadm/app/images/images_test.go

```diff
@@ -73,3 +75,46 @@ func TestGetCoreImage(t *testing.T) {
 		}
 	}
 }
+
+func TestGetAllImages(t *testing.T) {
```

Comment:
Could you add a test for: 
 
ci/latest
ci-cross/latest

here. 

I believe there should be other tests in a different location that are similar.

Reply:
Do you want to test that when a user specifies `--kubernetes-version ci/latest` and `ci-cross/latest` that the correct CIImageRepository gets set on the config?

If that's the case we should put it under the test for `config.NormalizeKubernetesVersion` function (which I see is only partially tested so I will add a few test cases).

Reply:
I'm ok with manual verification that the two targets translate correctly right now, but we need todo the cleanup work in 1.12 then.